vimrc_main

```diff
@@ -514,57 +514,140 @@ noremap Q! :q!<CR>
 nmap <silent> ,cq :cclose<CR>
 nmap <silent> ,co :copen<CR>
 
-" With tmux-plugins/vim-tmux-focus-events, vim will load up a file on focus so write the file when switching away
-let g:tmux_navigator_save_on_switch = 1
 
-""" LINTING (ALE)
-let g:ale_fixers = {
-\   'ruby': ['rubocop'],
-\   'javascript': ['prettier', 'eslint'],
-\   'json': ['prettier', 'eslint'],
-\   'elixir': ['mix_format'],
-\}
-let g:ale_linters = {
-\  'javascript': ['eslint'],
-\  'ruby': ['ruby','rubocop','reek'],
-\  'erb': ['erb'],
-\  'slim': ['slim-lint'],
-\  'sass': ['sass-lint'],
-\   'elixir': ['elixir-ls'],
-\}
-
-" \  'commitmessage': ['gitlint'],
-
-
-" let g:ale_javascript_prettier_eslint_options = 'something can go here'
-let g:ale_fix_on_save = 1
-
-" highlight clear ALEErrorSign " otherwise uses error bg color (typically red)
-" highlight clear ALEWarningSign " otherwise uses error bg color (typically red)
-" hi Error   cterm=bold gui=bold
-" hi Warning cterm=bold gui=bold
-" hi link ALEErrorSign    Error
-" hi link ALEWarningSign  Warning
-" highlight ALEErrorSign ctermbg=NONE ctermfg=red
-" let g:ale_sign_error = '✘' " could use emoji
-let g:ale_sign_error = '•' " could use emoji
-" let g:ale_sign_warning = '•' " could use emoji
-let g:ale_sign_warning = '💩' " could use emoji
-let g:ale_statusline_format = ['X %d', '? %d', '']
-" %linter% is the name of the linter that provided the message
-" %s is the error or warning message
-let g:ale_echo_msg_format = '%linter% says %s'
-let g:ale_sign_column_always = 1
-let g:ale_ruby_rubocop_options = '--except Lint/Debugger -D'
-let g:ale_lint_on_text_changed='normal' " only lint when exiting insert and going into normal mode
-let g:ale_lint_delay = 250
-let g:ale_set_highlights = 1
-
-" Map keys to navigate between lines with errors and warnings.
-nnoremap <leader>an :ALENextWrap<cr>
-nnoremap <leader>ap :ALEPreviousWrap<cr>
+" " TextEdit might fail if hidden is not set.
+" TODO Jon, this conflicts with line 479, not sure what the implications are
+" set hidden
```

Comment:
@jondkinney I noticed that hidden was set to something else, not sure what the right choice is.

Reply:
Sounds like hidden is a fine choice. No issues from me to switch that to the new default. Just curious, what's requiring/suggesting that?